workflow/tests/hpxml_translator_test.rb

```diff
@@ -538,6 +538,7 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml)
       next if err_line.include? 'DetailedSkyDiffuseModeling is chosen but not needed as either the shading transmittance for shading devices does not change throughout the year'
       next if err_line.include? 'View factors not complete'
       next if err_line.include?('CheckSimpleWAHPRatedCurvesOutputs') && err_line.include?('WaterToAirHeatPump:EquationFit') # FIXME: Check these
+      next if err_line.include?('For object = Coil:Cooling:DX:MultiSpeed,') # FIXME: The warning is not descriptive
```

Comment:
Need to address FIXME still.

Reply:
The warning is like that on [previous comment](https://github.com/NREL/OpenStudio-HPXML/pull/1244#issuecomment-1467016019) Is there a better way to describe it?

Reply:
Ideally, it seems like we would concatenate all the messages below the warning line to be part of the `err_line` variable, so that we could check against any part of the entire warning message. Do you want to take a stab at that?

Reply:
sure I will give it a try